opacus/grad_sample/grad_sample_module.py

```diff
@@ -257,6 +275,11 @@ def remove_hooks(self) -> None:
             delattr(self, "autograd_grad_sample_hooks")
             delattr(self._module, "autograd_grad_sample_hooks")
 
+        # Remove functorch hooks
+        for _module_name, module in trainable_modules(self._module):
+            if not type(module) in self.GRAD_SAMPLERS:
+                delattr(module, "ft_compute_sample_grad")
```

Comment:
Shouldn't this be done for all layers when force_functorch is True?

Reply:
Done. I'm thinking, we can also just check dynamically if the attribute is there and remove it?

Reply:
Thanks. That works too.